fdbserver/Status.actor.cpp

```diff
@@ -2377,10 +2376,22 @@ ACTOR static Future<JsonBuilderObject> blobWorkerStatusFetcher(
 
 		wait(waitForAll(futures));
 
+		state int totalRanges = 0;
 		for (auto future : futures) {
 			if (future.get().present()) {
 				auto latestTrace = future.get().get();
 				totalRanges += latestTrace.getInt("NumRangesAssigned");
+
```

Comment:
Also including the ranges assigned by worker in each worker's status would be good to do too.
Otherwise this looks great to me!

Reply:
Fixed. thank you for your comments!